phenotips/patient-network
# PN-161: Notifications does not handle contacts for remote matches correctly; also PN-164; PN-165; PN-166; PN-167; PN-169; PN-170; PN-171; PN-173; PN-178

To test PN-161 build in order:
- `phenotips` branch `rm-refactor` on top of milestone-5 with -Pquick
- `patient-network` branch `PN-161` with -Pquick
- `remote-matching` branch `PN-161`
- `phenomecentral`
Create a group with email info. Create and approve new user with email info. Create a new patient as admin, group and as a new user with same candidate gene and any phenotype. Check that `Contact` column for `Similar cases` section is filled in correctly.
Check that matching notification table has contact info filled in correctly.
